### ISSUE TITLE
Somtimes the app is not working and not giving any notifications

### ISSUE THREAD

**Issue by rafraph** [tom-james-watson/breaktimer-app#99]
Somtimes the app is not working and not giving any notifications about breaks.
I tried to `quit` and relaunch and I also tried to disable and enable but it is not working.
I think the only way to fix it is by restart the pc.
I'm working on win 10

**Comment by tom-james-watson** [comment]
We'd need a bit more information. Is the app actually running? Is it in the system tray? Does it show in the task manager?

**Comment by rafraph** [comment]
The app is running and I can see it in the system tray and also in task manager.
It works good all previous week, but today it just not working. And now, really now when I try to describe the problem I remember that this is Sunday, and maybe this is the reason. And I checked it and indeed  this was the problem. Since by default Sunday is not a working day but here is Israel we work on Sunday, so I changed it in setting and it start working like charm :-)
Thanks for your great tool. It really help me. I will recommend it on my Blog.
